unit_tests/src/test_client_connection.cpp

```diff
@@ -333,8 +332,9 @@ TEST_CASE("#7155: report underlying error info from server on failed connection"
             ix::client_connection{env.rodsHost, 8080, {env.rodsUserName, env.rodsZone}};
         }
         catch (const irods::exception& e) {
-            // Assertion fails with including the embedded errno value.
-            CHECK(e.code() == (USER_SOCK_CONNECT_ERR - ECONNREFUSED));
+            // Extract the iRODS error code, ignoring the embedded errno value.
```

Comment:
Is there any value in asserting the errno value? Is it important?

Reply:
I'm not sure. It's trivial to add, so will so. Someone may find it valuable in the future.

Reply:
Done.